hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/analysis/HoodieAnalysis.scala

```diff
@@ -102,7 +103,7 @@ case class HoodieResolveReferences(sparkSession: SparkSession) extends Rule[Logi
 
   def apply(plan: LogicalPlan): LogicalPlan = plan resolveOperatorsUp  {
     // Resolve merge into
-    case MergeIntoTable(target, source, mergeCondition, matchedActions, notMatchedActions)
+    case mergeInto@MergeIntoTable(target, source, mergeCondition, matchedActions, notMatchedActions)
```

Comment:
maybe better keep style like spark。  case mergeInto @ MergeIntoTable(target, source, mergeCondition, matchedActions, notMatchedActions)

Reply:
yes, will fix this format.